src/appengine/handlers/cron/fuzzer_weights.py

```diff
@@ -100,17 +100,17 @@ def _coverage_formatter(query_format, dataset):
 # included for the same reason.
 TIMEOUT_SPECIFICATION = QuerySpecification(
     adjusted_weight=0.50,
-    threshold=0.80,
+    threshold=0.95,
     query_format=GENERIC_QUERY_FORMAT.format(field_name='timeout_count'),
     formatter=_past_day_formatter,
     reason='frequent timeouts')
 
-# Fuzzers which are crashing frequently may not be making full use of their
-# allotted time for fuzzing, and may end up being more effective once the known
-# issues are fixed.
+# Fuzzers which are crashing on almost every run are not making full use of
+# their allotted time for fuzzing, and may end up being more effective once the
+# known issues are fixed.
 CRASH_SPECIFICATION = QuerySpecification(
     adjusted_weight=0.50,
-    threshold=0.90,
+    threshold=0.99,
```

Comment:
Why not 0.95 for everything - STARTUP_CRASH_SPECIFICATION, SLOW_UNIT_SPECIFICATION, TIMEOUT_SPECIFICATION, OOM_SPECIFICATION, CRASH_SPECIFICATION

Reply:
I'm not opposed to that but I don't think it would do as good a job of addressing the problem this change was meant to solve. I was hoping to address the cases we discussed yesterday where weights were being sporadically adjusted from day to day for pairings on the border of one of the thresholds. This .99 looks a bit odd in particular but this one in particular has a lot of cases that really do crash every time, and this does a better job of filtering out some of the ones that fall close to the border (this has quite a few > .9, where most of the rest tend not to).